### ISSUE TITLE
CT: [neurites] Keep the column titles fixed so they don't scroll with the rest of the window

### ISSUE THREAD

**Issue by drdiek** [Hippocampome-Org/php#468]

**Comment by nmsutton** [comment]
@drdiek @Nikhilkoneru I have been able to create a version of the matrix that has this feature. I noticed that some of the synaptic probabilities code has been recoded to save all echo statements into an array for saving in json. Thank you Nikhil for doing this. I tried to convert my updated code into the new json system and it seem to work on my local computer but not when I uploaded it to the site for some reason. The matrix does not seem to form right as seen here: [json version](http://www.hippocampome.org/phpdev/synapse_probabilities.php). I also uploaded a version without json that seems to work right and is here: [non json version](http://www.hippocampome.org/phpdev/synapse_probabilities_no_json.php). I probably am going to need help figureing out how to get the non-json version into working correctly as a json version.

**Comment by nmsutton** [comment]
@drdiek @Nikhilkoneru I tried to restore the working version of the synapse probabilities pages by uploading the last version of the files put on Github before I added my updated version with the fixed column but still the matrix is having the same problem. Either I don’t know how to fully get the json version working or there was some form of mismatch with the files version from Github. Either way that is likely to be the problem with the json not working right overall. I created a synapse_probabilities_with_json.php file in the base phpdev directory to show my fixed header updated version that was combined with Nikhol’s json code.
